torchgeo/datasets/western_usa_live_fuel_moisture.py

```diff
@@ -204,8 +201,6 @@ def __init__(
         input_features: list[str] = all_variable_names,
         transforms: Callable[[dict[str, Any]], dict[str, Any]] | None = None,
         download: bool = False,
-        api_key: str | None = None,
```

Comment:
Does this need a versionchanged, since those arguments were removed?

Reply:
There is versionadded and versionchanged but no versionremoved, so I haven't been adding this to other docstrings.